buildprocess/configureWebpack.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/*global require*/
```

Comment:
Yeah - all of these `/* global require */ things should be superfluous after you've told .eslintrc to allow it